docs/quickstart-minikube.md

```diff
@@ -231,7 +218,7 @@ WordPress to Kubernetes and OSBA will handle provisioning an Azure MySQL databas
 and binding it to our WordPress installation.
 
 ```
-$ helm install azure/wordpress --name quickstart
+$ helm install azure/wordpress --name osba-quickstart --namespace osba-quickstart
```

Comment:
> because helm will set the resource group to the same name as namespace

Practically speaking yes. But that's a feature (or quirk) of _our charts_ specifically.

> I'm not clear on why you changed the name of the wordpress installation though? Was there something wrong with quickstart? Do they have to match?

They don't have to, but I took a guess that requiring readers to mentally juggle "quickstart" (the release) and "osba-quickstart" (the namespace and, implicitly, the resource group) would probably lead to some mistakes.